### DISCUSSION TITLE
multi-GPU

### DISCUSSION THREAD

**Discussion by bigplanewangyue** [sarchlab/mgpusim#67] [Q&A] [answered]
Is there any way to build a multi-GPU environment with this simulator? I may not have a basic knowledge of go

**Comment by syifan** [comment, marked as answer]
The default configuration supports multi-GPU simulation. You can execute with something like `-gpus=1,2,3,4` or `-unified-gpus=1,2,3,4` to support multi-GPU execution. 

Go is very easy to learn.

**Reply by bigplanewangyue** [reply]
Thanks for your answer ！！！
